instrumentation/servlet/servlet-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/Servlet3Singletons.java

```diff
@@ -24,14 +24,6 @@
 public final class Servlet3Singletons {
   private static final String INSTRUMENTATION_NAME = "io.opentelemetry.servlet-3.0";
 
-  public static final VirtualField<Servlet, MappingResolver.Factory>
-      SERVLET_MAPPING_RESOLVER_FACTORY =
-          VirtualField.find(Servlet.class, MappingResolver.Factory.class);
-
-  public static final VirtualField<Filter, MappingResolver.Factory>
-      FILTER_MAPPING_RESOLVER_FACTORY =
-          VirtualField.find(Filter.class, MappingResolver.Factory.class);
```

Comment:
This change isn't necessary, but it seems odd that these are duplicated.  I combined them, but if it's intentional I can revert this change.